src/labels/CustomLabels.labels

```diff
@@ -13828,7 +14452,7 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Resource Marketing Engagement label in Get Started Resources section</shortDescription>
-        <value>Marketing/Engagement Learning Path</value>
+        <value>Engagement</value>
```

Comment:
This should say "Engagement Learning Path".

Reply:
In the slack channel we decided to shorten that to Engagment due to the lenght of the string in the translations. Can you confirm it @ameyersalesforce?

Reply:
It should be "Engagement Learning Path" instead of "Marketing/Engagement Learning Path"